typescript/introduction/tsc-intro/get-started-with-typescript.md

```diff
@@ -0,0 +1,91 @@
+---
+author: kapnobatai136
+type: normal
+category: must-know
+aspects:
+  - introduction
+links:
+  - '[What is a Compiler?](https://en.wikipedia.org/wiki/Compiler){website}'
+  - '[The TypeScript Compiler](https://www.npmjs.com/package/typescript){website}'
+  - '[Download Node.js](https://nodejs.org/en/download/){website}'
+
+---
+
+# Get started with TypeScript
```

Comment:
I've noticed that some headlines use title case and some don't. we should choose one and stay consistent